styles/lichess/catppuccin.user.css

```diff
@@ -0,0 +1,567 @@
+/* ==UserStyle==
+@name lichess Catppuccin
```

Comment:
I think Lichess would be capitalized here?

Reply:
```suggestion
@name Lichess Catppuccin
```